src/Raven.Client/Documents/Changes/DatabaseChanges.cs

```diff
@@ -620,7 +639,8 @@ private async Task DoWork(string nodeTag)
 
                 try
                 {
-                    await TimeoutManager.WaitFor(TimeSpan.FromSeconds(1), _cts.Token).ConfigureAwait(false);
+                    timerInSec = Math.Min(timerInSec * 2, 60);
```

Comment:
where do we set it back to `1` if we successfully connected?

Reply:
Added